assembly/runtime/collections/persistentDeque.ts

```diff
@@ -2,20 +2,49 @@ import { collections } from "../collections";
 import { storage } from "../storage";
 
 /**
-* A deque class that implements a persistent bidirectional queue.
-*/
+ * This class is one of several convenience collections built on top of the `Storage` class
+ * exposed by the NEAR platform.  It implements queue -- a persistent bidirectional queue
+ * (aka double-ended queue or deque)
+ *
+ * To create a deque
+ *
+ * ```ts
+ * let dq = PersistentDeque<string>("q")  // choose a unique prefix per account
+ * ```
+ *
+ * To use the deque
+ *
+ * ```ts
+ * dq.pushFront(value)
+ * dq.popBack()
+ * ```
+ *
+ * IMPORTANT NOTE:
```

Comment:
Maybe instead have a link to the the index?

Reply:
for important notes like this i'm in favor of repetition.  it's worse for maintenance but this is unlikely to change without a major overhaul to the design of storage so very low frequency change or no change